src/BorrowBookControl.java

```diff
@@ -5,23 +5,23 @@
 	
 	private BorrowBookUI ui;
 	
-	private library L;
-	private member M;
+	private library libary; //variable name should be in camle case form and as well as meaningful(L to Libarary)

```

Comment:
variable name should be in camle case form and as well as meaningful(L to Libarary)